lib/util/ui.js

```diff
@@ -1,5 +1,29 @@
+import PhoneNumber from 'awesome-phonenumber'
 import { Children, isValidElement, cloneElement } from 'react'
 
+/**
+ * @param {*} string the string to test.
+ * @returns true if the string is null or of zero length.
+ */
+export function isBlank (string) {
+  return !(string && string.length)
+}
+
+/**
+ * Formats a phone number according to the structure and format in phoneFormatOptions.
+ * @param {*} rawNumber The raw number (e.g. +155555555) to process
+ * @param {*} phoneFormatOptions The phoneFormatOptions from the configuration.
+ * @return The formatted number, or the raw number if no region code was provided.
+ */
+export function formatPhoneNumber (rawNumber, phoneFormatOptions) {
```

Comment:
In mongoDB, I updated the phoneNumber field to be removed and the resulting null value for phoneNumber caused this method to throw an uncaught error.

Reply:
No longer an issue with use of react-phone-number-input.